status/profile_plots.py

```diff
@@ -147,7 +149,7 @@ def iter_deployments():
     Iterates over all of the GliderDAC deployments and returns the dictionary
     containing the deployment attributes.
     '''
-    url = 'http://data.ioos.us/gliders/providers/api/deployment'
+    url = 'https://gliders.ioos.us/providers/api/deployment'
     response = requests.get(url)
```

Comment:
Should add a timeout here in the event the API doesn't respond.
Also consider using `response.raise_for_status` for errors.